incubator/airflow/templates/deployments-scheduler.yaml

```diff
@@ -0,0 +1,121 @@
+          - /usr/local/git/git-clone.sh
+          args:
+          - "{{ .Values.dags.git.url }}"
+          - "{{ .Values.dags.git.ref }}"
+          - "/dags"
+          volumeMounts:
+          - name: git-clone
+            mountPath: /usr/local/git
+          - name: dags-data
+            mountPath: /dags
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}-scheduler
+          image: {{ .Values.airflow.image.repository }}:{{ .Values.airflow.image.tag }}
+          imagePullPolicy: {{ .Values.airflow.image.pull_policy}}
+          envFrom:
+          - configMapRef:
+              name: "{{ template "airflow.fullname" . }}-env"
+          env:
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "airflow.fullname" . }}
+                  key: postgresUser
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "airflow.fullname" . }}
+                  key: postgresPassword
+            - name: REDIS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "airflow.fullname" . }}
+                  key: redisPassword
+          volumeMounts:
+          {{- if .Values.persistence.enabled }}
+            - name: dags-data
+              mountPath: {{ .Values.dags.path }}
+          {{- else if .Values.dags.init_container }}
+            - name: dags-data
+              mountPath: {{ .Values.dags.path }}
+            - name: scripts
+              mountPath: /usr/local/scripts
+          {{- end }}
+          args:
+            - "bash"
+            - "-c"
+          {{- if and ( .Values.dags.init_container ) ( .Values.dags.init_container.install_requirements ) }}
+            - >
+              echo 'waiting 10s...' &&
+              sleep 10 &&
+              echo 'installing requirements...' &&
+              /usr/local/scripts/install-requirements.sh &&
+              echo "executing initdb" &&
+              airflow initdb &&
+              echo "executing scheduler" &&
+              airflow scheduler -n {{ .Values.airflow.scheduler_num_runs }} {{- if .Values.airflow.scheduler_do_pickle }} -p {{- end }}
+          {{- else }}
+            - >
+              echo 'waiting 10s...' &&
+              sleep 10 &&
+              echo "executing initdb" &&
+              airflow initdb &&
+              echo "executing scheduler" &&
+              airflow scheduler -n {{ .Values.airflow.scheduler_num_runs }} {{- if .Values.airflow.scheduler_do_pickle }} -p {{- end }}
+          {{- end }}
+      volumes:
+        - name: dags-data
+        {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ .Values.persistence.existingClaim | default (include "airflow.fullname" .) }}
+        {{- else }}
+          emptyDir: {}
+        {{- end }}
+        {{- if .Values.dags.init_container }}
```

Comment:
`{{- if .Values.dags.init_container }}` should be `{{- if .Values.dags.init_container.enabled }}`, i guess.